operator/deploy/crd.yaml

```diff
@@ -30,11 +30,335 @@ spec:
     listKind: KialiList
     plural: kialis
     singular: kiali
+  preserveUnknownFields: false
   scope: Namespaced
-  subresources:
-    status: {}
-  version: v1alpha1
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
+    - name: v1alpha1
+      served: true
+      storage: true
+      subresources:
+        status: {}
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                installation_tag:
+                  type: string
+                istio_component_namespaces:
+                  type: object
+                  additionalProperties: true
+                istio_namespace:
+                  type: string
+                api:
+                  type: object
+                  properties:
+                    namespaces:
+                      type: object
+                      properties:
+                        exclude:
+                          type: array
+                          items:
+                            type: string
+                          default:
+                            - istio-operator
+                            - kube.*
+                            - openshift.*
+                            - ibm.*
+                            - kiali-operator
+                    label_selector:
+                      type: string
+                apidocs:
+                  type: object
+                  properties:
+                    annotations:
+                      type: object
+                      properties:
+                        api_spec_annotation_name:
+                          type: string
+                          default: kiali.io/api-spec
+                        api_type_annotation_name:
+                          type: string
+                          default: kiali.io/api-type
+                auth:
+                  type: object
+                  properties:
+                    strategy:
+                      type: string
+                      enum:
+                        - login
+                        - openshift
+                        - anonymous
+                    ldap:
+                      type: object
+                      properties:
+                        ldap_base:
+                          type: string
+                        ldap_bind_dn:
+                          type: string
+                        ldap_group_filter:
+                          type: string
```

Comment:
if we ARE going to put defaults in this schema, then we need to set this default which is `"(cn=%s)"`